dart/tool/service.md

```diff
@@ -37,6 +37,7 @@ The Service Protocol uses [JSON-RPC 2.0][].
 	- [getVersion](#getversion)
 	- [getVM](#getvm)
 	- [pause](#pause)
+  - [reloadSources](#reloadSources)
```

Comment:
Possibly a tab issue? @johnmccutchan (this file is copied from the latest from the sdk repo)